simple/msg_rma.c

```diff
@@ -203,10 +180,10 @@ static int run_test(void)
 	}
 	clock_gettime(CLOCK_MONOTONIC, &end);
 
-	if (machr)
-		show_perf_mr(transfer_size, iterations, &start, &end, 1, g_argc, g_argv);
+	if (opts.machr)
+		show_perf_mr(opts.transfer_size, opts.iterations, &start, &end, 1, opts.argc, opts.argv);
 	else
-		show_perf(test_name, transfer_size, iterations, &start, &end, 1);
+		show_perf(test_name, opts.transfer_size, opts.iterations, &start, &end, 1);
```

Comment:
If the opts are global, the shared code could handle this check.  The apps would just need to call show_perf().  Though I think I'd keep the show_perf API as-is.